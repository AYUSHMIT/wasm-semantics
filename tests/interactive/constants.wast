i32.const 3
(i32.const 5)
i64.const 71
f32.const 3.245
(f64.const 3.234523)
( (f64.const 1.21460644e+52) )
(f64.const 1.6085927714e-321)
(f64.const 1.63176601e-302)

// Below examples do not work with current float parser
// (f64.const 0x1.da21c460a6f44p+52)
// (f64.const 0x1.60859d2e7714ap-321)
// (f64.const 0x1.e63f1b7b660e1p-302)
