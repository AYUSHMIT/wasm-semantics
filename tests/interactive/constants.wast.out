<generatedTop>
  <k>
    .
  </k>
  <stack>
    < f64 > 1.631766009999999999999999999999999999999999999999999999999999999999999998E-302p237x19 : < f64 > 1.608592771399999999999999999999999999999999999999999999999999999999999998E-321p237x19 : < f64 > 1.214606440000000000000000000000000000000000000000000000000000000000000000E52p237x19 : < f64 > 3.234522999999999999999999999999999999999999999999999999999999999999999993E0p237x19 : < f32 > 3.244999999999999999999999999999999999999999999999999999999999999999999997E0p237x19 : < i64 > 71 : < i32 > 5 : < i32 > 3 : .Stack
  </stack>
</generatedTop>
